### PR TITLE
Mat 6579 adding email sender

### PR DESCRIPTION
**Pull request by gregory-akins**
## PR Template

Jira Ticket: [MAT-6579](https://jira.cms.gov/browse/MAT-6579)
(Optional) Related Tickets:

### Summary

Added email handling for error messages.  Also fixed issue with tests where coverage wasn't being reported correctly. Stil have some test coverage to add for the most recent changes, but getting this PR out and will fix that as a followup since I have another story to complete for this to be completely done.

Leaving my email as the destination email so 1) the email doesn't get delivered to the customers until it's formatted correctly, & 2) So that I can maybe see some of the messages that get reported to have a better idea of what type of messages will be expected.

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included
* [X] No extemporaneous files are included (i.e. Complied files or testing results)
* [X] This PR is into the **correct branch**.
* [X] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/library/logging.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import chalk from "chalk";
+
+export default class Logging {
+  // public static log = (args:any)=> this.info(args);
```
Comment (jkotanchik-SB) [inline lambda-function/src/utils/library/logging.ts]: Commented out duplicate line.

File: lambda-function/src/utils/library/logging.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import chalk from "chalk";
```
Comment (jkotanchik-SB) [inline lambda-function/src/utils/library/logging.ts:1]: Never heard of chalk, looks neat!

File: lambda-function/tests/unit/utils/mailservice.test.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    nodemailer.createTransport.mockReturnValue({ sendMail: sendMailMock, verify: () => {} });
+
+    const mailService: MailService = new MailService();
+    const info: SMTPTransport.SentMessageInfo = await mailService.sendMail("dev@example.com", "Error Message");
+    expect(info).toBeDefined();
+
+    expect(sendMailMock).toHaveBeenCalled();
+  });
+
+  it.skip("Calls SMTPClient, transporter verify returns an error", async () => {
```
Comment (jkotanchik-SB) [inline lambda-function/tests/unit/utils/mailservice.test.ts:26]: Is this skip expected?

File: lambda-function/src/utils/library/logging.ts (hunk 1 above)
Comment (gregory-akins) [reply lambda-function/src/utils/library/logging.ts]: That must have changed because of running "format". I didn't make any intentional changes to that file

File: lambda-function/tests/unit/utils/mailservice.test.ts (hunk 3 above)
Comment (gregory-akins) [reply lambda-function/tests/unit/utils/mailservice.test.ts:26]: Yeah.. for the time being.  Cleaning up in next story

**Review by jkotanchik-SB — APPROVED** [review]
